app/config/tstvhq/rooms.json

```diff
@@ -382,14 +382,12 @@
         {
             "id": "tstHQ",
             "name": "Infernal Theater",
-            "type": "IFRAME",
-            "capacity": 100,
+            "type": "JITSI",
+            "capacity": 20,
             "muteRoom": false,
             "hideVideo": false,
             "locked": false,
-            "description": "The crushed velvet seats recline! Are they emerald or black? Hard to tell in the dark.",
-            "iframeOptions": {
-                "src": "https://player.twitch.tv/?channel=LEDflashing&parent=estate.tstvhq.com"
+            "description": "The crushed velvet seats recline! Are they emerald or black? Hard to tell in the dark."
             },
```

Comment:
this bracket needs to be deleted

Reply:
here there is definitely an extra brace. it should be deleted, and the comma moved to the end of the previous line

Reply:
'k